packages/perpswap/src/contracts/vault.rs

```diff
@@ -19,12 +19,45 @@ pub struct InstantiateMsg {
     pub markets_allocation_bps: HashMap<String, u16>,
 }
 
+/// Denomination of USDC token
+#[derive(serde::Serialize, serde::Deserialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "snake_case")]
+pub enum UsdcDenom {
+    /// CW20 USDC
+    CW20(Addr),
+    /// IBC USDC
+    IBC(String),
```

Comment:
What's the difference between an IBC and native USDC? I think these two can be combined into a single `Native`.